src/System.Linq.Queryable/tests/packages.config

```diff
@@ -0,0 +1,22 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="System.Collections" version="4.0.10-beta-22405" targetFramework="portable-net45+win" />
+  <package id="System.IO" version="4.0.10-beta-22405" targetFramework="portable-net45+win" />
+  <package id="System.Linq" version="4.0.0-beta-22703" targetFramework="portable-net45+win" />
+  <package id="System.Linq.Expressions" version="4.0.10-beta-22703" targetFramework="portable-net45+win" />
+  <package id="System.Linq.Queryable" version="4.0.0-beta-22703" targetFramework="portable-net45+win" />
```

Comment:
The System.Linq.Queryable.Tests.csproj should be using a P2P reference to the System.Linq.Queryable.csproj, rather than pulling in System.Linq.Queryable's NuGet package.